### PR TITLE
helmfile 0.60.1

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
relates to https://github.com/Homebrew/homebrew-core/pull/39648

**Comment by chenrui333** [comment]
```
01:11:40 cp -pR /private/tmp/d20190514-41890-coiqio/helmfile-0.60.1/. /private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1
01:11:40 chmod -Rf +w /private/tmp/d20190514-41890-coiqio
01:11:40 ==> go build -ldflags -X main.Version=v0.60.1 -o /usr/local/Cellar/helmfile/0.60.1/bin/helmfile -v github.com/roboll/helmfile
01:11:40 tmpl/context_tmpl.go:5:2: cannot find package "github.com/Masterminds/sprig" in any of:
01:11:40 	/usr/local/Cellar/go/1.12.5/libexec/src/github.com/Masterminds/sprig (from $GOROOT)
01:11:40 	/private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1/src/github.com/Masterminds/sprig (from $GOPATH)
01:11:40 state/create.go:10:2: cannot find package "github.com/imdario/mergo" in any of:
01:11:40 	/usr/local/Cellar/go/1.12.5/libexec/src/github.com/imdario/mergo (from $GOROOT)
01:11:40 	/private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1/src/github.com/imdario/mergo (from $GOPATH)
01:11:40 state/state.go:23:2: cannot find package "github.com/tatsushid/go-prettytable" in any of:
01:11:40 	/usr/local/Cellar/go/1.12.5/libexec/src/github.com/tatsushid/go-prettytable (from $GOROOT)
01:11:40 	/private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1/src/github.com/tatsushid/go-prettytable (from $GOPATH)
01:11:40 cmd/cmd.go:8:2: cannot find package "github.com/urfave/cli" in any of:
01:11:40 	/usr/local/Cellar/go/1.12.5/libexec/src/github.com/urfave/cli (from $GOROOT)
01:11:40 	/private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1/src/github.com/urfave/cli (from $GOPATH)
01:11:40 helmexec/exec.go:11:2: cannot find package "go.uber.org/zap" in any of:
01:11:40 	/usr/local/Cellar/go/1.12.5/libexec/src/go.uber.org/zap (from $GOROOT)
01:11:40 	/private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1/src/go.uber.org/zap (from $GOPATH)
01:11:40 helmexec/exec.go:12:2: cannot find package "go.uber.org/zap/zapcore" in any of:
01:11:40 	/usr/local/Cellar/go/1.12.5/libexec/src/go.uber.org/zap/zapcore (from $GOROOT)
01:11:40 	/private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1/src/go.uber.org/zap/zapcore (from $GOPATH)
01:11:40 tmpl/context_funcs.go:5:2: cannot find package "gopkg.in/yaml.v2" in any of:
01:11:40 	/usr/local/Cellar/go/1.12.5/libexec/src/gopkg.in/yaml.v2 (from $GOROOT)
01:11:40 	/private/tmp/helmfile-20190514-41890-1kjl2b2/helmfile-0.60.1/src/gopkg.in/yaml.v2 (from $GOPATH)
01:11:40 
01:11:40 ==> Formula
01:11:40 Path: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/helmfile.rb
```

**Comment by chenrui333** [comment]
Relates to this commit, https://github.com/roboll/helmfile/commit/8f030d5eabc9b7ad7a8d16b6c384bbc8518a1e6f#diff-f949e2d81c8076ebbf8af38fcbb72c1f, they moved to `go modules` in 0.55, we need to adjust the formula. :)

**Comment by chenrui333** [comment]
I will fix the build issue.

**Comment by chenrui333** [comment]
Issue should be fixed now

**Comment by chenrui333** [comment]
```
00:14:54 ==> brew style helmfile
00:14:58 ==> FAILED
00:14:58 == /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/helmfile.rb ==
00:14:58 C: 39: 14: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
00:14:58 
00:14:58 1 file inspected, 1 offense detected
```

**Comment by chenrui333** [comment]
I cannot reproduce the issue locally.
```
helmfile-0.60.1 [$]
$ brew style helmfile

1 file inspected, no offenses detected
```

**Comment by chenrui333** [comment]
@BrewTestBot build me please

File: Formula/helmfile.rb [hunk 1]
```diff
@@ -34,7 +36,7 @@ def install
     - name: test
     EOS
     system Formula["kubernetes-helm"].opt_bin/"helm", "init", "--client-only"
-    output = '"stable" has been added to your repositories'
+    output = ""
```
Comment (fxcoudert) [inline Formula/helmfile.rb]: Match with empty string is not a good test at all, since it always passes I think.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/helmfile.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/helmfile.rb]: yeah, you are right, I did realize that 😆

Comment (chenrui333) [reply Formula/helmfile.rb]: I will fix it.

Comment (chenrui333) [reply Formula/helmfile.rb]: It looks like there is no output for this line, `shell_output("#{bin}/helmfile -f helmfile.yaml repos")`

Comment (zbeekman) [reply Formula/helmfile.rb]: then a new test is needed and we must validate that the new behavior (having no output) is actually valid, and not indicative of a problem.

**Comment by chenrui333** [comment]
Now the release is `https://github.com/roboll/helmfile/releases/tag/v0.64.2`, I will adopt the lessons from here to a new PR.

**Pull request closed without merging**